docs/stories/from-sphinx-to-mkdocs.md

```diff
@@ -182,13 +205,13 @@ files into markdown code snippets.
 
 While there are at least two solutions for including content with [mkdocs], they
 both are problematic because some decisions that were made regarding how
-relative paths are computed, something that breaks badly when you try to build a
+relative paths are computer, something that breaks badly when you try to build a
 single site out of multiple repositories. I am even wondering if this issue is
-fixable as it might touch the core concepts on which [mkdocs] is built.
+fixable as it might touch the core concepts on which [mkdocs] is build.
 
 It seems that mkdocs does not provide plugin authors with a way to get the path
 of the current document and instead it provides two paths, the
-`docs_path: docs/` and the configuration path. Not only this but it states
+`docs_path: docs/` and the path configuration path. Not only this but it states
```

Comment:
Hmm, looks like you committed old contents instead? This is not a spelling mistake but was corrected previously as well.